docs/docs/04-configuration/01-general.md

```diff
@@ -1,2 +1,25 @@
 # General
-Document any configuration that doesnt fit into model or auth providers. Ideally i'd like to see most if not all env vars we use in our render deployments documented. This doesn't need to be a hand holding guide, just give a decent idea of what each one does. 
\ No newline at end of file
+
+The Otto8 server is configured via environment variables. The following configuration is available:
```

Comment:
It's a nit, but are we calling this `Otto` or `Otto8`? I've seen it called both ways now in the docs.

Reply:
I've done a quick search in the docs and I believe I have made the name consistently `Otto8`.